rules-reviewed/eap6/jboss-eap5/jboss-eap5-xml.windup.xml

```diff
@@ -237,7 +236,8 @@
                 <xmlfile matches="//*[local-name()='jboss']" in="jboss.xml"/>
             </when>
             <perform>
-                <classification title="JBoss EJB Deployment Descriptor (prior to AS 7 / EAP 6)" effort="5">
+                <!-- Depending on the project, this should be set to severity="mandatory", effort="1". "ResolveJBossLegacyEjbXmlRuleProvider" classification use to duplicate this rule. -->
```

Comment:
but this means in every case a change/addition in `jboss-ejb3.xml` file, or i am wrong?